hs_swat_modelinstance/models.py

```diff
@@ -0,0 +1,550 @@
+            hsterms_model_input_rainfall_time_step.text = self.model_input.rainfall_time_step
+            hsterms_model_input_simulation_time_step = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}simulationTimeStep' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_simulation_time_step.text = self.model_input.simulation_time_step
+            hsterms_model_input_watershed_area = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}watershedArea' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_watershed_area.text = self.model_input.watershed_area+ ' km2'
+            hsterms_model_input_number_of_subbasins = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}numberOfSubbasins' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_number_of_subbasins.text = self.model_input.number_of_subbasins
+            hsterms_model_input_number_of_HRUs = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}numberOfHRUs' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_number_of_HRUs.text = self.model_input.number_of_HRUs
+            hsterms_model_input_DEM_resolution = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}DEMResolution' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_DEM_resolution.text = self.model_input.DEM_resolution + ' m'
+            hsterms_model_input_DEM_source_name = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}DEMSourceName' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_DEM_source_name.text = self.model_input.DEM_source_name
+            hsterms_model_input_DEM_source_URL = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}DEMSourceURL' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_DEM_source_URL.text = self.model_input.DEM_source_URL
+            hsterms_model_input_landUse_data_source_name = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}landUseDataSourceName' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_landUse_data_source_name.text = self.model_input.landUse_data_source_name
+            hsterms_model_input_landUse_data_source_URL = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}landUseDataSourceURL' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_landUse_data_source_URL.text = self.model_input.landUse_data_source_URL
+            hsterms_model_input_soil_data_source_name = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}soilDataSourceName' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_soil_data_source_name.text = self.model_input.soil_data_source_name
+            hsterms_model_input_soil_data_source_URL = etree.SubElement(hsterms_model_input_rdf_Description, '{%s}soilDataSourceURL' % self.NAMESPACES['hsterms'])
+            hsterms_model_input_soil_data_source_URL.text = self.model_input.soil_data_source_URL
```

Comment:
Most of this enormous block of repetitive code could be reduced significantly by looping over the element names